packages/components/src/InputPhoneNumber/InputPhoneNumber.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { InputMask } from "./InputMask";
+import { CommonFormFieldProps, FormField, FormFieldProps } from "../FormField";
+
+interface InputPhoneNumberProps
+  extends Omit<CommonFormFieldProps, "align">,
+    Pick<
+      FormFieldProps,
+      | "autocomplete"
+      | "onEnter"
+      | "onFocus"
+      | "onBlur"
+      | "validations"
+      | "readonly"
+      | "prefix"
+      | "suffix"
+    > {
+  readonly value: string;
+  readonly onChange: (value: string) => void;
+
+  /**
+   * Shows a "required" validation message when the component is left empty.
+   */
+  readonly required?: boolean;
+}
+
+export function InputPhoneNumber({
+  required,
+  ...props
+}: InputPhoneNumberProps) {
+  const { placeholder, validations } = props;
+  const pattern = "(***) ***-****";
+
+  return (
+    <InputMask pattern={pattern} strict={false}>
+      <FormField
+        {...props}
+        type="tel"
+        validations={{
+          required: {
+            value: Boolean(required),
+            message: `${placeholder || "This"} is required`,
```

Comment:
I like this in general, no notes to change it. I also look forward to new and wonderful ways that this breaks unintentionally. (E.g. "Please enter your phone number is required")

Reply:
ha! I was gonna say "good thing we have rules!" but I don't think we have one.

Reply:
This is a fine thing for us to be opinionated about, so having labels be a noun of the "thing" being inputted is ok. I wonder if we have anywhere we can/should communicate that?

Reply:
yeah, considering placeholder/label is a pretty big part of input, I think it would be rare to see this (_pun intended_). I think we can start with design and see what they think